tests/vanilla/dependency.test.tsx

```diff
@@ -94,7 +94,7 @@ it('correctly handles the same promise being returned twice from an atom getter
   await expect(store.get(derivedAtom)).resolves.toBe('Asynchronous Data')
 })
 
-it('keeps atoms mounted between recalculations', async () => {
+it('keeps sync atoms mounted between recalculations', async () => {
```

Comment:
This is the planned breaking change in v2.12.0.

@yuneco Will you be able to check if this breaks your app?

`npm i https://pkg.csb.dev/pmndrs/jotai/commit/b4bb8a8d/jotai`